### ISSUE TITLE
optional Int fields show NaN even if no values were entered

### ISSUE THREAD

**Issue by omar-dulaimi** [prisma/studio#637]
prisma               : 2.17.0
@prisma/client       : 2.17.0
Current platform     : debian-openssl-1.1.x
Query Engine         : query-engine 3c463ebd78b1d21d8fdacdd27899e280cf686223 (at node_modules/@prisma/engines/query-engine-debian-openssl-1.1.x)
Migration Engine     : migration-engine-cli 3c463ebd78b1d21d8fdacdd27899e280cf686223 (at node_modules/@prisma/engines/migration-engine-debian-openssl-1.1.x)
Introspection Engine : introspection-core 3c463ebd78b1d21d8fdacdd27899e280cf686223 (at node_modules/@prisma/engines/introspection-engine-debian-openssl-1.1.x)
Format Binary        : prisma-fmt 3c463ebd78b1d21d8fdacdd27899e280cf686223 (at node_modules/@prisma/engines/prisma-fmt-debian-openssl-1.1.x)
Studio               : 0.353.0

```
model Item {
  deleteReason       Int?
  soldBy             Int?
}
```
![Screenshot from 2021-02-22 00-53-44](https://user-images.githubusercontent.com/11743389/108721067-3dbffa80-752a-11eb-9f03-a5f71673a5a7.png)

Clicking/Tabbing produce this issue.

Confirmed issue on both Web and Electron versions.

**Comment by madebysid** [comment]
@omar-dulaimi Just to confirm, this is a visual thing only, right? I'll make it so `null` numbers show up as `null`!

**Comment by omar-dulaimi** [comment]
@madebysid Yeah I think only visually because it didn't actually save NaNs.

**Comment by madebysid** [comment]
Hey @omar-dulaimi This is now fixed in Prisma CLI `2.18.0-dev.48`. You should be able to `npm install prisma@dev` to get this immediately. However, I'd recommend updating to the next stable version when it comes out, since `dev` versions are mostly for internal use and may contain unannounced breaking changes.

This will also make it to stable v2.18.0 scheduled for this Tuesday!
